components/brave_rewards/core/ledger_impl.h

```diff
@@ -382,28 +390,7 @@ class LedgerImpl : public mojom::Ledger {
     }
   }
 
-  template <typename T>
-  T GetOption(const std::string& name) {
-    T value;
-
-    if constexpr (std::is_same_v<T, bool>) {
-      ledger_client_->GetBooleanOption(name, &value);
-    } else if constexpr (std::is_same_v<T, int32_t>) {
-      ledger_client_->GetIntegerOption(name, &value);
-    } else if constexpr (std::is_same_v<T, double>) {
-      ledger_client_->GetDoubleOption(name, &value);
-    } else if constexpr (std::is_same_v<T, std::string>) {
-      ledger_client_->GetStringOption(name, &value);
-    } else if constexpr (std::is_same_v<T, int64_t>) {
-      ledger_client_->GetInt64Option(name, &value);
-    } else if constexpr (std::is_same_v<T, uint64_t>) {
-      ledger_client_->GetUint64Option(name, &value);
-    } else {
-      static_assert(base::AlwaysFalse<T>, "Unsupported type!");
-    }
-
-    return value;
-  }
+  bool GetIsBitFlyerRegion();
```

Comment:
Can callers just do `ledger_->client()->IsBitflyerRegion()` instead?

Reply:
They could, but then they'd have to change its syntax to the out-parameter one (which is why we have helpers in `LedgerImpl`).